### ISSUE TITLE
the pre-trained generator is missing 

### ISSUE THREAD

**Issue by ShangyinGao** [leftthomas/SRGAN#65]
The paper says the SRResNet is first trained and then used to initialize the generator. 
In your code, it seems like the generator does not use the pre-trained SRResNet.
Am I correct?

**Comment by leftthomas** [comment]
@ShangyinGao first trained on what dataset?

**Comment by avinabsaha** [comment]
The paper mentions training using MSE loss first for few epochs, without directly starting to train with Adversarial and Content losses. Thus this mplementation is slightly different. @leftthomas can you share Set 5 and Set14  benchmarking scores that you got, with this approach? Btw, Thanks for such a great work.

**Comment by leftthomas** [comment]
@avinabsaha run `test_benchmark.py`, then you will get the results.

**Comment by avinabsaha** [comment]
The paper mentions training using MSE loss first for few epochs, without directly starting to train with Adversarial and Content losses.
